Tests/Application/bin/console

```diff
@@ -19,7 +18,12 @@ $env = $input->getParameterOption(array('--env', '-e'), getenv('APP_ENV') ?: 'de
 $debug = getenv('APP_DEBUG') !== '0' && !$input->hasParameterOption(array('--no-debug', '')) && $env !== 'prod';
 
 if ($debug) {
-    Debug::enable();
+    // Clean up when sf 4.3 support is removed
+    if (Kernel::VERSION_ID < 40411) {
```

Comment:
This ist incorrect the both classes beneath are not part of the http kernel component so the Kernel Version can not be used for this. Instead you should go with class exist in the errorhandler class.